index.bs

```diff
@@ -175,12 +180,21 @@ spec: mediaqueries-4; urlPrefix: https://drafts.csswg.org/mediaqueries-4/#
 
       <pre class='idl'>
         enum MediaEncodingType {
-          "todo",
+          "record",
+          "transmission"
         };
       </pre>
 
       <p>
-        TODO
+        A {{MediaEncodingConfiguration}} can have one of two types:
+        <ul>
+          <li><dfn for='MediaEncodingType' enum-value>record</dfn> is used to
+          represent a configuration for recording of media, e.g. using
+          {{MediaRecorder}} as defined in [[mediastream-recording]].</li>
+          <li><dfn for='MediaEncodingType' enum-value>transmission</dfn> is used
+          to represent a configuration meant to be transmitted over electronic
+          means (e.g. using {{RTCPeerConnection}}).</li>
```

Comment:
I copy-pasted-modified the [`MediaDecodingConfiguration`](https://github.com/WICG/media-capabilities/blob/master/index.bs#L160) entry:

       A {{MediaDecodingConfiguration}} has two types:
        <ul>
          <li><dfn for='MediaDecodingType' enum-value>file</dfn> is used to
          represent a configuration that is meant to be used for a plain file
          playback.</li>
          <li><dfn for='MediaDecodingType' enum-value>media-source</dfn> is used
          to represent a configuration that is meant to be used for playback of
          a {{MediaSource/MediaSource}} as defined in the [[media-source]]
          specification.</li>
        </ul>
      </p>

Should I modify both or...?